dbt_subprojects/dex/models/dex_info.sql

```diff
@@ -198,6 +198,8 @@ FROM (VALUES
     , ('levinswap', 'Levinswap', 'Direct', 'levinswap')
     , ('sparkdex', 'SparkDEX', 'Direct', 'SparkDexAI')
     , ('blazeswap', 'BlazeSwap', 'Direct', 'blazeswapdex')
+    , ('unagi', 'Unagi', 'Direct', 'UnagiProtocol')
+    , ('izumi_finance', 'iZUMi Finance', 'Direct', 'izumi_Finance')
```

Comment:
```suggestion
```

Reply:
we already have a row for `izumi` above in the file. we should stick to one entry and be consistent. this means using `izumi` throughout the code in PR (file names, schema names, project name column, etc etc.)